crates/bevy_math/src/cubic_splines.rs

```diff
@@ -440,36 +484,34 @@ impl<P: Point> CubicNurbs<P> {
         Ok(Self {
             control_points,
             weights,
-            knot_vector,
+            knots,
         })
     }
 
-    /// Generates a uniform knot vector that will generate the same curve as [`CubicBSpline`].
+    /// Generates uniform knots that will generate the same curve as [`CubicBSpline`].
     ///
-    /// "Uniform" means that the difference between two knot values next to each other is the same
-    /// through the entire knot vector.
+    /// "Uniform" means that the difference between two subsequent knots is the same.
     ///
-    /// Will return `None` if there are less than 4 control points
-    pub fn uniform_knot_vector(control_points: usize) -> Option<Vec<f32>> {
+    /// Will return `None` if there are less than 4 control points.
+    pub fn uniform_knots(control_points: usize) -> Option<Vec<f32>> {
         if control_points < 4 {
             return None;
         }
         Some(
-            (0..Self::knot_vector_length(control_points))
+            (0..Self::knots_len(control_points))
                 .map(|v| v as f32)
                 .collect(),
         )
     }
 
-    /// Generates an open uniform knot vector, which makes the ends of the curve pass through the
+    /// Generates open uniform knots, which makes the ends of the curve pass through the
     /// start and end points.
     ///
-    /// The knot vector will have a knot with multiplicity of 4 at the end and start and elements
-    /// in the middle will have a difference of 1. "Multiplicity" means that there are N
-    /// consecutive elements that have the same value.
+    /// The start and end knots have multiplicity 4, and intermediate knots have multiplicity 0 and
+    /// difference of 1. "Multiplicity" means that there are N consecutive elements that have the same value.
```

Comment:
```suggestion
    /// The start and end knots have multiplicity 4, and intermediate knots have multiplicity 0 and
    /// difference of 1.
```
As you have explained what Multiplicity is in the main doc comment of `CubicNurbs`, I don't think there is need to have this here.